src/zendoo/mcTestCall.cpp

```diff
@@ -5,6 +5,7 @@
 #include <iostream>
 #include <cassert>
 #include <string>
+#include "gtest/libzendoo_test_files.h"
```

Comment:
Why do you need this ?

Reply:
I took SAMPLE_FIELD from this file for the default value.

Reply:
But you can safely pass NULL to the zendoo_create_cert_test_proof function